saleor/checkout/calculations.py

```diff
@@ -278,6 +279,7 @@ def _fetch_checkout_prices_if_expired(
                     address,
                 )
             except TaxEmptyData as e:
+                _get_checkout_base_prices(checkout, checkout_info, lines)
```

Comment:
this is nonblocking comment, but I'm wondering why this function is called `_get...` when it is rather a setter for a checkout?

Reply:
Yeah it triggered me as well, just forgot about it. I'll change it.